modules/certificate/certificate.graphql.js

```diff
@@ -1,25 +1,51 @@
-const certificateType = `
+const certificateTypes = `
   type Certificate {
     _id: ID!
-    createdBy: ID!
     name: String!
     value: Int!
-    isUsed: Boolean!
-    isActive: Boolean!
+    createdBy: User
+    ownedBy: User
+    email: String
+    isUsed: Boolean
+    isActivated: Boolean!
+    isExpired: Boolean
     dateStart: Date
     dateEnd: Date
+    isActive: Boolean @deprecated(reason: "use isActivated and isExpired fields.")
```

Comment:
Not quite sure that here is necessity for directives. I think it's proper to use directives in case when there are a lot of apps which use this app. As here we have single connection between front-end and backend i think it's more proper to delete this field and add some hotfix to frontend part.

Reply:
Deleted